src/dynamic_honey_badger/dynamic_honey_badger.rs

```diff
@@ -122,11 +377,24 @@ where
 
     /// Cast a vote to change the set of validators.
     pub fn vote_for(&mut self, change: Change<N>) -> Result<Step<C, N>> {
+        info!(
+            "{:?}, {}being a validator, is voting for {:?}",
+            self.netinfo.our_id(),
+            if self.netinfo.is_validator() {
+                ""
+            } else {
+                "not "
+            },
+            change
+        );
+        if let Change::Add(ref id, ..) = change {
+            self.nodes_being_added.insert(id.clone());
+        }
```

Comment:
Should us _casting_ the vote really be the trigger to add the node to `nodes_being_added`, rather than the vote getting the majority?